pom.xml

```diff
@@ -141,6 +141,11 @@
         <artifactId>proto-google-common-protos</artifactId>
         <version>${google.common-protos.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <version>${google.core.version}</version>
+      </dependency>
```

Comment:
Did you mean?
```xml
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>google-cloud-core</artifactId>
        <version>${google.core.version}</version>
        <scope>test</scope>
      </dependency>
```

Reply:
> Did you mean?
> 
> ```
>       <dependency>
>         <groupId>com.google.cloud</groupId>
>         <artifactId>google-cloud-core</artifactId>
>         <version>${google.core.version}</version>
>         <scope>test</scope>
>       </dependency>
> ```

if you want to used `ServiceOptions.getDefaultProjectId()` in this repo then we have to expliccitly add it,same i did it for othe client as well.

Reply:
But scope can be test right? Is it necessary to bundle with client?

Reply:
just try with adding `<scope>test</scope>`, If works then you are all set.

Reply:
jeff suggest me in datacatalog,see here same thing i did it for it.
https://github.com/athakor/java-datacatalog/blob/master/pom.xml#L141

https://github.com/googleapis/java-datacatalog/blob/master/google-cloud-datacatalog/pom.xml#L77

Reply:
Ok, I understand your point 👍